stock_inventory_discrepancy/models/stock_quant.py

```diff
@@ -23,9 +24,24 @@ class StockQuant(models.Model):
         compute="_compute_discrepancy_threshold",
     )
     has_over_discrepancy = fields.Boolean(
-        compute="_compute_has_over_discrepancy",
+        compute="_compute_has_over_discrepancy", search="_search_has_over_discrepancy"
     )
 
+    def _search_has_over_discrepancy(self, operator, value):
+        if operator not in ["=", "!="]:
```

Comment:
if you support "!=" you should consider de operator in the above computation

Reply:
that's true, thanks